semcore/dropdown/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.13.0] - 2023-11-29
+
+### Changed
+
+- Deprecated some values of `interaction` property.
+
+## [4.12.0] - 2023-11-28
+
+### Added
+
+- `interaction='click'` by default. Pass `focusMaster=true` for `Popover` if `interaction='click'`.
```

Comment:
I understood almost nothing from this note 😅

Reply:
removed, we don't need it